src/quo2/components/inputs/input/view.cljs

```diff
@@ -100,6 +101,7 @@
            (cond-> {:style                  (style/input colors-by-status small? @multiple-lines?)
                     :accessibility-label    :input
                     :placeholder-text-color (:placeholder colors-by-status)
+                    :keyboard-appearance    (colors/theme-colors :light :dark override-theme)
```

Comment:
hi, `:light` and `:dark` are not colors, Please use `(if (colors/dark?) :dark :light)`

Reply:
Or maybe we should rename function (or create a new one) and put in utils etc.
Something like `(utils/theme-value light-theme-value dark-theme-value override-theme)`
@J-Son89 @ilmotta wdyt?

Reply:
or put in `quo2.theme`?

Reply:
cc: @flexsurfer

Reply:
I have seen the function being used rarely for things other than color, like here: https://github.com/status-im/status-mobile/blob/0c16ffcc21fc3b89d8d26ab25a74656e7066a716/src/status_im2/contexts/chat/composer/style.cljs#L12.

Despite the name, the function returns whatever we pass in the props based on the theme/override-theme. But I think renaming/making a new function can make more sense.

Reply:
I agree with @ulisesmac and @Parveshdhull, it's not a good idea to use this function even if it works here. It's intention is to set colors. Let's create some other helper as suggested, perhaps something more generic in name 👍

Reply:
I agree with @J-Son89 and others. And I also like @Parveshdhull's suggestion for a utility function outside the `colors` ns to get any value based on a theme.

Reply:
I agree with you guys. I will create a generic utility function and update the PR.

Reply:
@Parveshdhull @ulisesmac @J-Son89  @ilmotta I have added a `theme/theme-value` function. Basically, it's the same as `colors/theme-colors` function with a different name. Let me know what you think of this implementation.